mr/src/main/java/org/apache/iceberg/mr/InputFormatConfig.java

```diff
@@ -73,6 +74,10 @@ private InputFormatConfig() {
   public static final String SNAPSHOT_TABLE = "iceberg.snapshots.table";
   public static final String SNAPSHOT_TABLE_SUFFIX = "__snapshots";
 
+  public static final String CATALOG_TYPE_TEMPLATE = CATALOG_NAME + ".%s.type";
```

Comment:
I think we reused here the CATALOG_NAME constant, which was supposed to be something else.

Maybe `CATALOG_CONFIG_PREFIX_`?

The value could be `iceberg.catalog.` or simply just omit?

Reply:
Done.